hydroflow_lang/src/graph/flat_graph.rs

```diff
@@ -34,8 +35,10 @@ pub struct FlatGraph {
     /// Spanned error/warning/etc diagnostics to emit.
     pub(crate) diagnostics: Vec<Diagnostic>,
 
+    /// What variable name each graph node belongs to (if any).
+    pub(crate) node_varnames: SparseSecondaryMap<GraphNodeId, Rc<Ident>>,
```

Comment:
I'm curious when it's worth using an Rc ref and when to copy. I assume `Ident` is small?

Reply:
Hmm, yeah probably better and definitely simpler to just `Clone`